presto-hive/pom.xml

```diff
@@ -14,6 +14,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <parquet.version>1.10.0</parquet.version>
```

Comment:
Why don't you define this in the root pom.xml? We also re-define this in presto-parquet module below.

Reply:
Done